chapters/evaluation/index.rst

```diff
@@ -0,0 +1,154 @@
+--------------------
+
+For any search system, measuring system performance is critical, and so it is for Jina.
+
+Generally, evaluation is about computing a difference between an output and a desired "ideal" results,
+users generally place an emphasis on evaluating final document ranking results using information retrieval metrics,
+such as Precision, Recall, mAP or nDCG.
+However, it ignores the fact that a search system is often composed by multiple components,
+whereas evaluation on the final results hardly reveal useful insights about the system. Jina allows the user
+to evaluate any part of the system with arbitrary metrics.
+
+
+Before you start
+-------------------
+
+We expect you have a clean Python 3.7/3.8/3.9 (virtual) build.
+With Jina installed on your machine:
+
+.. highlight:: bash
+.. code-block:: bash
+
+    pip install -U jina
+
+
+Overview
+-----------------
+
+To achieve our objective, Jina has a family of :term:`Executor` named :term:`Evaluator`.
+These evaluators are meant to capture Documents from any part of the Flow in order to evaluate them.
+
+As a new type of :term:`Executor`, evaluators inspect documents from the request and comparing them with ground truth.
+This executors can be wrapped in a :term:`Pod` as any other kind of executor and placed anywhere in the :term:`Flow`.
+They tend to be placed after the :term:`Pod` applying the transformation that wants to be evaluated by the specific `Evaluator`.
+
+In order to be able to Evaluate the performance of a transformation applied to a Document by any part of a Jina
+:term:`Flow`, we need to know what the desired state of the :term:`Document` is. This desired state is called
+:term:`GroundTruth` and can be passed in Jina in every ``IndexRequest`` and ``SearchRequest``. This :term:`GroundTruth`
+in Jina is nothing else than another :term:`Document`.
+
+``IndexRequest`` and ``SearchRequest`` are formed by streams of pairs and `Documents` and `GroundTruths`. When no evaluation
+is involved, `GroundTruth` tends to be empty, however when an `Evaluation` pod is involved in the Flow, it will actually
+take the information of every `GroundTruth` to feed both `Document` and `GroundTruth` information to the evaluator :term:`Executor`
+
+The evaluation, in general, follows a two step approach: *extraction* and *evaluation*.
+Jina :term:`Driver` extracts document and groundtruth information from :term:`Protobuf`,
+and pass this information to the executor.
+The second steps happens inside the executor: evaluate the difference between these two documents into a number.
+Afterwards, the :term:`Driver` will add the results of the evaluation into the `evaluations` field of the `Document`.
+
+
+Since Evaluation tends to focus only on some small parts of the `Documents` (IDs of the matches when evaluating Rankers,
+embedding when evaluating Encoders), it is not needed for `GroundTruth` to contain more information from the `Document` than
+the one that will be used by the Evaluators.
```

Comment:
**Old**
it is not needed for `GroundTruth` to contain more information from the `Document` than
the one that will be used by the Evaluators.

**New**
`GroundTruth` does not need to contain more information from the `Document` than will be used by the Evaluator.